source-chargebee-native/source_chargebee_native/resources.py

```diff
@@ -34,92 +39,136 @@
     snapshot_resource,
     fetch_resource_page,
     fetch_resource_changes,
+    snapshot_associated_resource,
+    fetch_associated_resource_page,
+    fetch_associated_resource_changes,
 )
 
 
-# TODO(jsmith): revisit
-# PARENT_CHILD_RESOURCES: dict[str, tuple[str, str]] = {
-#     "contacts": ("customers", "id"),  # (parent_resource, parent_key_field)
-#     "subscriptions_with_scheduled_changes": ("subscriptions", "id"),
-# }
+COMMON_STANDALONE_FULL_REFRESH_RESOURCES = [
+    "site_migration_details",
+    "quote_line_groups",  # Requires Performance or Enterprise Chargebee subscription plan - will throw HTTP 404 if not enabled
```

Comment:
@williamhbaker Note that this will throw a 404 if a capture is set up to capture this binding and their account does not have this feature available (via the plan they are on) or enabled in their account.

Should this be handled in another way or should we allow the HTTP 404 to occur as long as we have it stated in the documentation?